ray-operator/controllers/ray/common/pod.go

```diff
@@ -809,7 +813,7 @@ func addWellKnownAcceleratorResources(rayStartParams map[string]string, resource
 		if !isCustomAcceleratorResourceAdded {
 			if rayResourceName, ok := customAcceleratorToRayResourceMap[resourceKeyString]; ok && !resourceValue.IsZero() {
 				if _, exists := resourcesMap[rayResourceName]; !exists {
-					resourcesMap[rayResourceName] = float64(resourceValue.Value())
+					resourcesMap[rayResourceName] = resourceValue.AsApproximateFloat64()
```

Comment:
Why change this to AsApproximateFloat64?

Reply:
To address this comment: https://github.com/ray-project/kuberay/pull/2425#discussion_r1792321453